src/Symfony/Bundle/WebServerBundle/WebServerBundle.php

```diff
@@ -11,8 +11,13 @@
 
 namespace Symfony\Bundle\WebServerBundle;
 
+use Symfony\Component\Console\Application;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 class WebServerBundle extends Bundle
 {
+    public function registerCommands(Application $application)
+    {
+        // noop
```

Comment:
This may be technically correct, but it looks super strange, mostly because Symfony is always so explicit.

Reply:
should be more explicit now